test/util.py

```diff
@@ -16,5 +16,17 @@ def check_art_dataset(dataset: DatasetWrapper):
         assert "Color Study" in image.name
     delete_dataset(dataset)
 
+
+def check_seg_dataset(dataset: DatasetWrapper):
+    """
+    Check if the test segmentation dataset has been uploaded correctly
+    """
+    assert dataset.name == "Nuclear image"
+    assert dataset.countChildren() == 1
+    for image in dataset.listChildren():
+        assert "Nuclear image" in image.name
+        # TODO: check segmentation here; using image.sizeOfRois() does not work
```

Comment:
Flagging that this still needs doing. There must be an API you can use if not that one.